docs/computing/lumi-vs-mahti.md

```diff
@@ -0,0 +1,55 @@
+
+To facilitate installing software on LUMI, the [EasyBuild tool](https://docs.lumi-supercomputer.eu/software/installing/easybuild/) is provided along with [installation recipes](https://github.com/Lumi-supercomputer/LUMI-EasyBuild-contrib) (EasyConfig files) using which you can install additional applications to your home or project directories. Additionally, a [container wrapper](https://docs.lumi-supercomputer.eu/software/installing/container_wrapper/) identical to the [Tykky tool](containers/tykky.md) is provided as a means to wrap installations inside an Apptainer/Singularity container. This is recommended especially for Conda and pip environments to alleviate the load on the parallel filesystem.
+
+If you have problems installing your software on LUMI-C, please [send a ticket to the LUMI user support team](https://lumi-supercomputer.eu/user-support/need-help/)!
+
+## Programming environment and software stacks
+
+The programming environment of LUMI-C is quite different compared to CSC supercomputers. LUMI comes with three alternative programming environments, namely Cray, GNU and AOCC. Each of the environments have their own compiler suites that become available upon loading the corresponding programming environment module. Moreover, two types of software stacks are offered, the CrayEnv and LUMI stacks. Please refer to the LUMI documentation for a detailed description of the [available collection of compiler suites](https://docs.lumi-supercomputer.eu/development/compiling/prgenv/) and [software stacks](https://docs.lumi-supercomputer.eu/computing/softwarestacks/) and how to swap between these.
+
+!!! Note
+    Irrespective of the loaded compiler suite, one noticeable difference concerning the LUMI-C programming environment is that it comes with compiler wrappers that replace commands commonly found on HPC systems such as Mahti. For example, commands for compiling MPI code like `mpicc`, `mpic++` and `mpif90` are unavailable as such. Instead, you should use the wrappers `cc`, `CC` and `ftn`, respectively. See the LUMI documentation for [more details on the available MPI wrappers](https://docs.lumi-supercomputer.eu/development/compiling/prgenv/#compile-an-mpi-program).
+
+## Disk areas
+
+Similar to CSC supercomputers, LUMI uses the Lustre parallel filesystem. However, there's currently no fast local disk such as on Puhti and Mahti-AI. The fast flash-based scratch space (LUMI-F) will be made available once the installation of LUMI-G is completed. Also, the object storage (LUMI-O) is not available yet. [See the LUMI documentation for more details](https://docs.lumi-supercomputer.eu/storage/).
```

Comment:
"Currrently no", I do not think it is planned either?

Reply:
Though I recently noticed Frontier has 2x2TiB in all nodes

Reply:
The or a? It is not Mahtis :)

Reply:
> "Currrently no", I do not think it is planned either?

Maybe I misunderstood. LUMI-F will be made available though, but I guess it is not a fast local disk, but a fast flash-based storage space? Is this what you mean?

Reply:
Exactly, it is flash based lustre file system, not local disk.  https://docs.lumi-supercomputer.eu/storage/parallel/lumif/